src/components/about/About.jsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import AboutImage from "../../assets/images/about-image.png";
+import "./about.scss";
+
+
+const About = () => {
+  return (
+    <div className="about">
+      <div className="about-container">
```

Comment:
Maybe this could be named something like an `about-top` instead container because it doesn't contain all the about page content. And this way you can wrap then description and the additional in one div called `about-bottom`.